pkg/repository/backend_postgres.go

```diff
@@ -917,11 +938,48 @@ func (c *PostgresBackendRepository) GetDeploymentByNameAndVersion(ctx context.Co
 	return &deploymentWithRelated, nil
 }
 
+// GetDeploymentByStubGroup retrieves the a deployment by name, version, and stub group
```

Comment:
minor typo: `retrieves the a deployment` should just be `retrieves the deployment`

Reply:
Fixed!